modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards_admin/js/MBEntriesManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {postForm} from 'frontend-js-web';
+
+export default function propsTransformer({
+	additionalProps: {
+		deleteEntriesCmd,
+		editEntryURL,
+		lockCmd,
+		trashEnabled,
+		unlockCmd,
+	},
+	portletNamespace,
+	...otherProps
+}) {
+	const form = document.getElementById(`${portletNamespace}fm`);
```

Comment:
If we wanted a local component variable, we would use `useRef`.

If that local variable is something that is calculated only once, on first render, we would use `useEffect`. In it, we would initiate our `ref`.

But, in this case, we should use neither. This is because DOM queries need to be checked on execution, not only on first render. This means we should have both queries and `if (form)` checks in each method.